src/components/GitmojiList/Toolbar/styles.module.css

```
@@ -16,6 +16,33 @@
   margin-right: 1rem;
 }
 
+.inputWrapper {
+  width: 100%;
+  margin-top: 0.357rem;
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.kbdWrapper {
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  padding: 0.425rem 1.5rem 0 0;
+}
+
+.kbd {
+  font-family: system-ui;
+  display: inline-flex;
+  align-items: center;
+  border: solid 1px;
+  padding: 0.5rem;
+  border-radius: 3px;
+  color: rgb(133, 133, 133);
+  border-color: rgb(133, 133, 133);
```

I don't like the color of the border, it seems very bold compared to the text. I think that `#bbb` looks much better.

---

also, you can merge `boder-color` and `border` into one property. 
```suggestion
  border: solid 1px #bbb;
```